pkgs/servers/gpsd/default.nix

```diff
@@ -104,7 +101,8 @@ stdenv.mkDerivation rec {
       diagnostic monitoring and profiling of receivers and feeding
       location-aware applications GPS/AIS logs for diagnostic purposes.
     '';
-    homepage = "http://catb.org/gpsd/";
+    homepage = "https://gpsd.gitlab.io/gpsd/index.html";
+    changelog = "https://gitlab.com/gpsd/gpsd/-/blob/release-${version}/NEWS";
```

Comment:
I've never seen meta.changelog before. Is there some tooling that uses it?

Reply:
I don't know about such tooling, but it's quite useful for human as well to quickly check what is changed, moreover it's not always that easy to find where is an official changelog.

It described here https://nixos.org/manual/nixpkgs/stable/#sec-standard-meta-attributes:

>  **changelog**
> 
>  A link or a list of links to the location of Changelog for a package. A link may use expansion to refer to the correct changelog version. Example: "https://git.savannah.gnu.org/cgit/hello.git/plain/NEWS?h=v${version}"

But it's not so widely used so far in nixpkgs:
```
$  grep --include=\*.nix -rnw nixpkgs/ -e "changelog =" | wc -l
304
```

Reply:
Nice. Thanks!